vpr/src/route/partition_tree.h

```diff
@@ -25,6 +25,19 @@ inline Side operator!(const Side& rhs) {
     return Side(!size_t(rhs));
 }
 
+/** Part of a net. Used by \ref DecompNetlistRouter. */
+class VirtualNet {
+  public:
+    /** The net in question. */
+    ParentNetId net_id;
+    /** Clipped bounding box. This is needed to enable decomposing a net multiple times.
+     * Otherwise we would need a history of side types and cutlines to compute the bbox. */
```

Comment:
What is a "clipped" bounding box?  Explain clipped.  Is this bounding box used to define the region of the chip within which routing resources may be used for this partial/decomposed net?  If so, say that.